src/ui/layout/constraints.md

```diff
@@ -2002,41 +2051,96 @@ BoxConstraints.tight(Size size)
      maxHeight = size.height;
 ```
 
-If you revisit [Example 2](#example-2) above,
-it tells us that the screen forces the red
-`Container` to be exactly the same size as the screen.
-The screen does that, of course, by passing tight
+If you revisit [Example 2](#example-2),
+the screen forces the red `Container` to be
+exactly the same size as the screen.
+The screen achieves that, of course, by passing tight
 constraints to the `Container`.
 
-A _loose_ constraint, on the other hand,
-sets the **maximum** width and height, but lets the widget
-be as small as it wants. In other words,
-a loose constraint has a **minimum** width and height
-both equal to **zero**:
+### Loose constraints
 
-```dart
-BoxConstraints.loose(Size size)
-   : minWidth = 0.0,
-     maxWidth = size.width,
-     minHeight = 0.0,
-     maxHeight = size.height;
-```
+A _loose_ constraint is one that has a minimum
+of zero and a maximum non-zero.
+
+Some boxes _loosen_ the incoming constraints,
+meaning the maximum is maintained but the
+minimum is removed, so the widget can have
+a **minimum** width and height both equal to **zero**.
 
-If you revisit [Example 3](#example-3), it tells us that the
-`Center` lets the red `Container` be smaller,
-but not bigger than the screen. The `Center` does that,
-of course, by passing loose constraints to the `Container`.
-Ultimately, the `Center`'s very purpose is to transform
-the tight constraints it got from its parent
+Ultimately, `Center`'s purpose is to transform
+the tight constraints it received from its parent
 (the screen) to loose constraints for its child
 (the `Container`).
 
+If you revisit [Example 3](#example-3), 
+the `Center` allows the red `Container` to be smaller,
+but not bigger than the screen.
+
+[`build`]: {{site.api}}/flutter/widgets/State/build.html
+[`RenderView`]: {{site.api}}/flutter/rendering/RenderView-class.html
+
+## Unbounded constraints
```

Comment:
I believe this is the section we want to link to from https://github.com/flutter/flutter/blob/c8e42b47f5ea8b5ff7bf2f2b0a2a8e765f1aa51d/packages/flutter/lib/src/rendering/flex.dart#L733. How do we do that?

Reply:
But one way to solve it is to set up a redirect from flutter.dev/layout to that page. I'd prefer that the short-url point to something more clear, like flutter.dev/layout-error, but this would work!